pkg/types/aws_resource_manager.go

```diff
@@ -89,4 +90,7 @@ type AWSResourceManagerFactory interface {
 	) (AWSResourceManager, error)
 	// IsAdoptable returns true if the resource is able to be adopted
 	IsAdoptable() bool
+	// RequeueOnSuccessSeconds returns true if the resource should be requeued after specified seconds
```

Comment:
java doc is out of date. Please fix it and everything else looks very good. 💯

Reply:
@vijtrip2 I'll fix on a followup (handling the TODO above) :)

Reply:
Thanks Jay.